apps/web/src/ee/clerk/providers/ClerkProvider.tsx

```diff
@@ -0,0 +1,52 @@
+import { PropsWithChildren } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { CLERK_PUBLISHABLE_KEY, IS_CLERK_AUTH_ENABLED } from '../../../config/index';
+import { ClerkProvider as _ClerkProvider } from '@clerk/clerk-react';
+import { useColorScheme } from '@novu/design-system';
+import { dark } from '@clerk/themes';
+
+const ClerkModalElement = {
+  modalContent: {
+    width: '80rem',
+    display: 'block',
+  },
+  cardBox: {
+    width: '100%',
+  },
+  rootBox: {
+    width: 'auto',
+  },
+};
+
+const localization = {
+  userProfile: {
+    navbar: {
+      title: 'Settings',
+      description: 'Manage your account settings',
+    },
+  },
+};
+
+export const ClerkProvider: React.FC<PropsWithChildren<{}>> = ({ children }) => {
```

Comment:
```suggestion
export const ExternalAuthProvider: React.FC<PropsWithChildren<{}>> = ({ children }) => {
```
Or something else non-vendor specific

Reply:
I see the point but I'm wondering if renaming this wouldn't cause more confusion since this is specifically ClerkProvider same as e.g. HelmetProvider, it providers just Clerk and nothing else. Because we also already have `EnterpriseAuthProvider`, adding `ExternalAuthProvider` could be confusing.

Reply:
I moved the `ClerkProvider` inside `AuthEnterpriseProvider` - this way we have just one auth provider for EE so it should be less confusing and its also vendor agnostic.